raster/r.topidx/r.topidx.html

```diff
@@ -56,8 +56,3 @@ <h2>AUTHORS</h2>
 Hydro Laboratory, Kyungpook National University, South Korea
 <p>
 Based on GRIDATB.FOR by Keith Beven.
-
```

Comment:
Line 43 `<h2>REFERENCES</h2>` section should be before `<h2>SEE ALSO</h2>` section.